internal/venice-common/src/main/java/com/linkedin/venice/authentication/jwt/ClientAuthenticationProviderToken.java

```diff
@@ -0,0 +1,50 @@
+package com.linkedin.venice.authentication.jwt;
+
+import com.linkedin.venice.authentication.ClientAuthenticationProvider;
+import com.linkedin.venice.utils.VeniceProperties;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.Map;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+
+public class ClientAuthenticationProviderToken implements ClientAuthenticationProvider {
+  private static final Logger log = LogManager.getLogger(ClientAuthenticationProviderToken.class);
+
+  private Map<String, String> headers;
+
+  public ClientAuthenticationProviderToken() {
+  }
+
+  public static ClientAuthenticationProviderToken TOKEN(String token) {
+    ClientAuthenticationProviderToken provider = new ClientAuthenticationProviderToken();
+    provider.headers = Collections.singletonMap("Authorization", "Bearer " + token);
+    return provider;
+  }
+
+  @Override
+  public void initialize(VeniceProperties veniceProperties) throws Exception {
+    String token = veniceProperties.getString("authentication.token", "");
```

Comment:
better to move it to the `ConfigKeys` class ?

Reply:
there was already the constant. I am now using it. Thanks